distribution/lib/Standard/Base/0.0.0-dev/src/Data/Read/Return_As.enso

```diff
@@ -0,0 +1,78 @@
+import project.Any.Any
+import project.Data.Text.Text
+import project.Data.Read.Many_Files_List.Many_Files_List
+import project.Data.Vector.Vector
+import project.Error.Error
+import project.Errors.Common.Type_Error
+import project.Errors.Illegal_Argument.Illegal_Argument
+import project.Function.Function
+import project.Metadata.Display
+import project.Metadata.Widget
+import project.Nothing.Nothing
+import project.Panic.Panic
+from project.Data.Boolean import Boolean, False, True
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Single_Choice
+
+polyglot java import org.enso.base.read.ReadManyReturnSPI
+
+private _get_known_return_classes -> Vector =
+    Vector.from_polyglot_array (ReadManyReturnSPI.get_types False)
+
+## A common interface that represents ways to return a list of files that have
+   been read.
+type Return_As
+    ## PRIVATE
+    Instance underlying
```

Comment:
Why this constructor isn't `private`?

Looks like a missed chance to [follow type classes blueprint](https://github.com/enso-org/enso/issues/5446#issuecomment-2425715747).

Reply:
The same reason as `Many_Files_List`.